src/main/java/ru/iopump/qa/allure/entity/ReportEntity.java

```diff
@@ -90,6 +90,10 @@ public boolean isFullUrl() {
         return checkUrl(url);
     }
 
+    public UUID getUuid() {
```

Comment:
It's redundant because there is Lombok `@Data` annotation on the class

Reply:
done